proposals/lip-0055.md

```diff
@@ -400,6 +400,24 @@ unsignedBlockHeaderSchema = {
 }
 ```
 
+
+#### Block Signature Calculation
+
+The block header signature is calculated using the `calculateBlockSignature` function. This function returns a bytes value with length `SIGNATURE_LENGTH_BYTES`.
+
+```python
+calculateBlockSignature():
+  generatorKey = validators.getValidatorAccount(block.header.generatorAddress).generatorKey
+  let secretKey be the private key corresponding to the public generatorKey
```

Comment:
Not important, but should we use "secret key" and "secretKey" (or "private key" and "privateKey") for consistency?

Reply:
good point, edited